circle.yml

```diff
@@ -3,22 +3,15 @@ machine:
     version: oraclejdk7
   environment:
     TERM: dumb
-    GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
-
-general:
-  artifacts:
-    - error-handling/build/libs
-    - http-clients/build/libs
-    - http-servers/build/libs
-    - ssl-config/build/libs
```

Comment:
As it stands, the artifact list was incomplete. Tests results and failures are collected automatically as far as I understand. Did you have another use for these?